stable/concourse/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: concourse
-version: 6.3.0
+version: 6.3.1
```

Hi @taylorsilva !

Would you mind bumping to `6.4.1` with the new PR that just got in? https://github.com/helm/charts/pull/14545

Thanks!

```suggestion
version: 6.4.1
```

---

Done 👍